Preview8

SeppPenner opened the pull request:
For me, this works as expected. I simply followed https://devblogs.microsoft.com/aspnet/asp-net-core-and-blazor-updates-in-net-core-3-0-preview-8/ and updated everything to preview8.

Fixes https://github.com/Joelius300/ChartJSBlazor/issues/57.

SeppPenner committed:
Updated one missing nuget package to preview8, did necessary changes for preview8.

SeppPenner committed:
Updated package reference from Microsoft.AspNetCore.Components.Browser to *.Web.

Joelius300 committed:
Add missing =, @functions->@code

Joelius300 commented:
There was a small error, you missed a = which broke the doughnut example. This is now fixed.  

I have also renamed `@functions` to `@code` because this is also related to upgrading the sdk and only affects the sample project.

SeppPenner commented:
> There was a small error, you missed a = which broke the doughnut example. This is now fixed.

Copy paste error I guess...

> I have also renamed @functions to @code because this is also related to upgrading the sdk and only affects the sample project.

I have adjusted this only in the readme and forgot it here.
